dll/win32/shell32/lang/da-DK.rc

```diff
@@ -1008,4 +1008,8 @@ BEGIN
     IDS_COPYTOTITLE "Select the place where you want to copy '%s'. Then click the Copy button."
     IDS_COPYITEMS "Copy Items"
     IDS_COPYBUTTON "Copy"
+    IDS_MOVETOMENU "Move to &folder..."
+    IDS_MOVETOTITLE "Select the place where you want to move '%s' to. Then click the Copy button."
```

Comment:
IDS_MOVETOTITLE "Wählen Sie den Ort, in den '%s' verschoben werden soll. Klicken Sie anschließend auf Verschieben."